tests/dummy/lib/eager-blog/addon/templates/post.hbs

```diff
@@ -0,0 +1,3 @@
+<div>
```

Comment:
Can we simplify this template to just be an `h4` or just `p`? Minor, but want the markup to be semantic.